### DISCUSSION TITLE
ESP32-S3 USB CDC

### DISCUSSION THREAD

**Discussion by Seeung** [nanoframework/Home#1502] [Q&A] [answered]
How to use USB CDC on ESP32-S3?

**Comment by alberk8** [comment]
The USB CDC is used by WireProtocol (default firmware based on devkit with two USB C).  At the moment I don't see any code that exposes that to nF.

**Comment by Seeung** [comment, marked as answer]
I found solution, USB CDC used as `COM1`.
Configuration USB CDC is in [binutils.ESP32.cmake](https://github.com/nanoframework/nf-interpreter/blob/e1e84139b3a0d74185772c47e938b6590183d9d8/CMake/binutils.ESP32.cmake#L725) file.
